### ISSUE TITLE
WebComponents / Toolbar more button issue

### ISSUE THREAD

**Issue by jpradelle** [nhn/tui.editor#2905]
## Describe the bug
When editor in used inside a WebComponent, click on toolbar more button doesn't open menu.
Similar issue than #2477

## To Reproduce
1. Put editor in a WebComponent with a limited width so toolbar more button is displayed
2. Click on More button
3. Menu doesn't open

jsfiddle demo: https://jsfiddle.net/jpradell/azvgw1cd/9/

## Expected behavior
More button menu should appear

## Desktop (please complete the following information):
 - OS: Debian Linux
 - Browser chrome
 - Version 102.0.5005.115

**Comment by jajugoguma** [comment]
Unfortunately, the editor does not work correctly within shadow roots. We will review it internally and fix it. thank you

**Comment by thomas-darling** [comment]
@jpradelle Any update on this?

We just ran into https://github.com/nhn/tui.editor/issues/2477, which appears to be related - closed by the stale bot, but very much still an issue.
We absolutely need this to work inside a shadow root, to avoid problems with conflicting CSS styles.

**Comment by jpradelle** [comment]
Yes PR was closed and not merged even if it fixed the issue. I switched to another project because of this issue and few other issues

**Comment by ps20developer** [comment]
@jajugoguma Any update on this?

We are having a similar issue in version 2.5.4, wysiwyg mode does not run properly with shadow dom.
The issues we have faced in shadow dom are

1. Press enter is not ok, will be added always on top, not at the cursor point
2. Backspace is not ok, it doesn't delete the character at the cursor point
3. Canc is not ok, deletes enter from top, not the character at the cursor point
4. Toolbar buttons are not clickable

We need this to work in shadow dom.
